torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -641,6 +641,52 @@ class TORCH_CUDA_CU_API LoadStoreOp : public Expr {
   Val* const in_ = nullptr;
 };
 
+
+// Convenience utility to initialize IterDomain's without having to sort through
+// all the default values. Intended to be used with IterDomain::IterDomain(IrBuilderPasskey IterDomainBuildArgs)
+class TORCH_CUDA_CU_API IterDomainBuilder{
+public:
+ // Match legacy constructor
+ IterDomainBuilder(Val* _start, Val* _extent);
+
+ // Grab all the parameters from id to set the IterDomainBuilder
+ IterDomainBuilder(const IterDomain* id);
+
+ // Resets defaults for rfactor, is padded dim, padded to size, and is mma
```

Comment:
Isn't the parallel type also considered a scheduling parameter?

Reply:
Yeah, good point.